TypeCobol/Compiler/Parser/CodeElementBuilder/CobolExpressionsBuilder.cs

```diff
@@ -815,81 +816,87 @@ private ConditionOperand CreateConditionOperand(CodeElementsParser.ConditionOper
 			return conditionOperand;
 		}
 
-		internal SyntaxProperty<RelationalOperator> CreateRelationalOperator(CodeElementsParser.RelationalOperatorContext context)
-		{
-			if(context.strictRelation() != null)
-			{
-				bool invertStrictRelation = false;
-				if (context.NOT() != null)
-				{
-					invertStrictRelation = true;
-				}
+		internal RelationalOperator CreateRelationalOperator(CodeElementsParser.RelationalOperatorContext context)
+        {
+            var notToken = ParseTreeUtils.GetFirstToken(context.NOT());
```

Comment:
No check on `context.NOT() != null` ?

Reply:
The check is done in `GetFirstToken()`, but it can be done before to avoid errors if the method is modified.